econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -0,0 +1,1038 @@
+                                      for key, value in props])
+
+            # TODO: enrich outcome logic for multi-class classification when that is supported
+            index = pd.MultiIndex.from_tuples([(i, outcome, res.feature_name, f"{lvl}v{res.feature_baseline}"
+                                                if res.feature_baseline is not None
+                                                else lvl)
+                                               for i in range(n)
+                                               for outcome in ["y0"]
+                                               for res in self._results
+                                               for lvl in res.feature_levels],
+                                              names=["sample", "outcome", "feature", "feature_value"])
+            for lvl in index.levels:
+                if len(lvl) == 1:
+                    if not isinstance(index, pd.MultiIndex):
+                        # can't drop only level
+                        index = pd.Index(["value"])
+                    else:
+                        index = index.droplevel(lvl.name)
+            return pd.DataFrame(to_include, index=index)
+
+        return self._summarize(summary=make_dataframe,
+                               get_inference=get_inference,
+                               props=props,
+                               expand_arr=expand_arr,
+                               drop_sample=False)  # dropping the sample dimension is handled above instead
+
+    def _dict_summary(self, get_inference, *, props, kind, drop_sample=False, expand_arr=False):
+        """
+        Summarizes results into a dictionary.
+
+        Parameters
+        ----------
+        get_inference : lambda
+            Method to get the relevant inference results from each result object
+        props : list of (string, string or lambda)
+            Set of column names and ways to get the corresponding values from the inference object
+        kind : string
+            The kind of inference results to get (e.g. 'global', 'local', or 'cohort')
+        drop_sample : boolean, default False
+            Whether to drop the sample dimension from each array
+        expand_arr : boolean, default False
+            Whether to add an initial sample dimension to the result arrays
+        """
+        def make_dict(props):
+            # should be serialization-ready and contain no numpy arrays
+            res = _get_default_specific_insights(kind)
+            res.update([(key, value.tolist()) for key, value in props])
+            return {**self._shared, **res}
+
+        return self._summarize(summary=make_dict,
+                               get_inference=get_inference,
+                               props=props,
+                               expand_arr=expand_arr,
+                               drop_sample=drop_sample)
+
+    def global_causal_effect(self, alpha=0.05):
```

Comment:
@vasilismsr I think this default for alpha was taken from the prototype, but we use different values for alpha in different places in the library; most of our inference methods use alpha=0.1; however grf and the _StatsModelsWrapper use alpha=0.05.  Should those places (and this class) be made to default to alpha=0.1 to be consistent with the rest of the package?

Reply:
0.05 should be the default thorughout
90% confidence is too low.
95% confidence is more standard

Reply:
Okay, in that case I'll do that in a separate change to come in the next release, but it will affect a *lot* of defaults.

Reply:
yeap makes sense